src/melee.cpp

```diff
@@ -992,7 +992,7 @@ float Character::get_dodge() const
 
     // Each dodge after the first subtracts equivalent of 2 points of dodge skill
```

Comment:
this comment should read different or be removed.
my tongue in cheek suggestion is `// got out of dodge`, but.